### DISCUSSION TITLE
[Feat]: Add ability to customise opacity percentages during fade

### DISCUSSION THREAD

**Discussion by freddieerg** [davidjerleke/embla-carousel#1025] [Ideas]
### Which variants of Embla Carousel are relevant to this feature request?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [X] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

I believe the current default implementation of embla-carousel-fade causes the currently selected slide to fade out at the inverse rate that the target slide is being faded in. ie, if you are half way through a drag, both slides are at 50% opacity. This results in the background of the container being visible, which may not always be the desired behaviour.

I would like to see if we could agree on a user option for this plugin that would allow some customisation of the opacities. My specific requirement is the ability to keep the outgoing slide at 100% until the incoming slide is also at 100%, thus avoiding any visibility of the container background.

### Additional Context

If we're able to come to some agreement on an implementation, I'm quite happy to create a PR for this.

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by freddieerg** [comment]
@davidjerleke were you able to take a look at this please? :)

**Reply by davidjerleke** [reply]
Hi @freddieerg,

Thanks for your question. No, I haven't had the time yet.
